azure-pipelines.yml

```diff
@@ -133,7 +133,7 @@ stages:
     pool: sonictest
     displayName: "kvmtest-t0-part1"
     timeoutInMinutes: 360
-
+    continueOnError: true
```

Comment:
is t0 not mandatory?

Reply:
t0-part 1 and t0-part2 are set as not mandatory,
kvmtest-t0 will check the results of part1 and part2, and it's mandatory